<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hadesvm::cereon::ResidentRamUnitEditor</class>
 <widget class="QWidget" name="hadesvm::cereon::ResidentRamUnitEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="startAddressLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start (hex):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startAddressLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sizeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Size:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sizeNumberOfUnitsLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeUnitComboBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startAddressLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hadesvm::cereon::ResidentRamUnitEditor</receiver>
   <slot>_onStartAddressLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeNumberOfUnitsLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hadesvm::cereon::ResidentRamUnitEditor</receiver>
   <slot>_onSizeNumberOfUnitsLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeUnitComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>hadesvm::cereon::ResidentRamUnitEditor</receiver>
   <slot>_onSizeUnitComboBoxCurrentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_onStartAddressLineEditTextChanged(QString)</slot>
  <slot>_onSizeNumberOfUnitsLineEditTextChanged(QString)</slot>
  <slot>_onSizeUnitComboBoxCurrentIndexChanged(int)</slot>
 </slots>
</ui>
